src/Aspire.Hosting/Kafka/KafkaServerResource.cs

```diff
@@ -20,20 +20,10 @@ public class KafkaServerResource(string name) : ContainerResource(name), IResour
     /// </summary>
     public EndpointReference PrimaryEndpoint => _primaryEndpoint ??= new(this, PrimaryEndpointName);
 
-    private ReferenceExpression ConnectionString =>
-        ReferenceExpression.Create(
-            $"{PrimaryEndpoint.Property(EndpointProperty.Host)}:{PrimaryEndpoint.Property(EndpointProperty.Port)}");
-
     /// <summary>
     /// Gets the connection string expression for Kafka broker for the manifest.
```

Comment:
(nit) not just for the manifest.

```suggestion
    /// Gets the connection string expression for the Kafka broker.
```